sqlite3_opt_math_functions.go

```diff
@@ -0,0 +1,15 @@
+// Copyright (C) 2019 Yasuhiro Matsumoto <mattn.jp@gmail.com>.
```

Comment:
@mattn What's the expectation for the copyright statement on new files?

Reply:
s/2019/2022/